docs/modules/ROOT/pages/index.adoc

```diff
@@ -3,3 +3,21 @@
 nfs-subdir-external-provisioner is a Commodore component to manage nfs-subdir-external-provisioner.
```

Comment:
Again, probably worth to note that this component is a replacement for nfs-client-provisioner, because upstream renamed their software